### PR TITLE
Fixed EETestCase, added Creaper

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+import org.jboss.hal.testsuite.finder.Application;
+import org.jboss.hal.testsuite.finder.FinderNames;
+import org.jboss.hal.testsuite.finder.FinderNavigation;
+import org.jboss.hal.testsuite.page.Navigatable;
+import org.jboss.hal.testsuite.util.ConfigUtils;
 
 /**
  * @author mkrajcov <mkrajcov@redhat.com>
  */
 @Location("#ee")
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java]: Please remove obsolete location to avoid confusion.

**Comment by honza-kasik** [comment]
Removed obsolete annotation from EEPage!

File: common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
-public class EEPage extends ConfigPage {
+public class EEPage extends ConfigurationPage implements Navigatable {
+
+    public void navigate() {
+        FinderNavigation navigation;
+        if (ConfigUtils.isDomain()) {
+            navigation = new FinderNavigation(browser, DomainConfigEntryPoint.class);
+            navigation.step(FinderNames.CONFIGURATION, FinderNames.PROFILES)
+                    .step(FinderNames.PROFILE, ConfigUtils.getDefaultProfile())
+                    .step(FinderNames.SUBSYSTEM, "EE");
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java]: This step is same for standalone as well as domain ;-)

File: common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+        } else {
+            navigation = new FinderNavigation(browser, StandaloneConfigEntryPoint.class);
+            navigation.step(FinderNames.CONFIGURATION, FinderNames.SUBSYSTEMS)
+                    .step(FinderNames.SUBSYSTEM, "EE");
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/EEPage.java]: This step is same for standalone as well as domain ;-)

**Comment by zebrik** [comment]
LGTM. Thx.

**Pull request merged**